### PR TITLE
Serge

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
https://git.dev.d2l/projects/RE/repos/serge-localize/pull-requests/113/overview

### CODE REVIEW

File: lang/locales/es-MX.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* This file is auto-generated. Do not modify directly. */
+/* eslint quotes: ["error", "double"] */
+
+export const esMx = {};
```
Comment (ctwomey1) [inline lang/locales/es-MX.js]: don't you need the keys in each of these or it blows up?

Comment (jstefaniuk-d2l) [reply lang/locales/es-MX.js]: I don't think so! But will check

Comment (jstefaniuk-d2l) [reply lang/locales/es-MX.js]: doesn't quite blow up, just any missing terms don't show, put a fix into the gulp script

File: lang/json/ar-SA.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (apalaniuk) [inline lang/json/ar-SA.json]: We can just let these be created by the Serge process, then add these languages when they're created the first time

**Pull request merged**